packages/grafana-ui/src/components/Gauge/Gauge.tsx

```diff
@@ -145,11 +147,15 @@ export class Gauge extends PureComponent<Props> {
   }
 
   renderVisualization = () => {
-    const { width, value, height, onClick, text, theme } = this.props;
+    const { width, value, height, onClick, text, theme, orientation } = this.props;
     const autoProps = calculateGaugeAutoProps(width, height, value.title);
+
+    // If the gauge is in vertical layout, we need to set the width of the gauge to the height of the gauge
+    const gaugeWidth = orientation === VizOrientation.Vertical ? `${autoProps.gaugeHeight}px` : '100%';
```

Comment:
Not confident on this - necessary for vertical mode to render somewhat correctly but scaling of value / label is off when resized

Without this change
<img width="1182" alt="Screenshot 2023-07-14 at 5 02 38 PM" src="https://github.com/grafana/grafana/assets/22381771/0c4e727e-885b-4221-bd39-93ed042e1b46">


With change / scaling issues (for vertical setting (laid out horizontally though...))



Uploading scaling.mov…